rust/ql/integration-tests/hello-workspace/summary.rust-project.expected

```diff
@@ -14,7 +14,7 @@
 | Macro calls - resolved | 2 |
 | Macro calls - total | 2 |
 | Macro calls - unresolved | 0 |
-| Taint edges - number of edges | 1471 |
+| Taint edges - number of edges | 1670 |
```

Comment:
I wonder if this row of output is something we should keep, or is it more bother than it's worth?

Reply:
It's nice to see the number go up. On the other hand, I haven't figured out how to run these locally so I've been updating them manually from the CIs output. Is there a way to run them locally?